src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/LogsPage/index.html.twig

```diff
@@ -52,11 +52,18 @@
           <div class="card-block row">
             <div class="card-text">
               <div class="form-group row">
-                {% set helptip = 'Enter "5" if you do not want to receive any emails.'|trans({}, 'Admin.Advparameters.Help') ~ ' ' ~ 'Emails will be sent to the shop owner.'|trans({}, 'Admin.Advparameters.Help') %}
-                {{ ps.label_with_help(('Minimum severity level'|trans), helptip, 'col-sm-2') }}
+                {% set helptip_severity = 'Leave empty if you do not want to receive any emails.'|trans({}, 'Admin.Advparameters.Help') ~ ' ' ~ 'Put in the following fields the recipients of these emails.'|trans({}, 'Admin.Advparameters.Help') %}
```

Comment:
Where does this wording will appear? Is it here? 

![image](https://user-images.githubusercontent.com/70583503/94661187-06487b80-0307-11eb-88c7-465b16e31247.png)


If yes, I think the word "field" should be singular because there is only one field.

Suggestion of wording : "Indicate the recipients of the email alerts in this field."

Reply:
Yes ! Thanks... but I'm not sure about you suggestion @LouiseBonnard can you check please ? It is ok for you ?

Reply:
Yes, Louise is supposed to double-check my wording suggestions ☺️ 

I just feel like "following fields" suggests that several fields have to be filled in, but I might be wrong.

Reply:
Indeed, there is just one field. Here is a slightly modified suggestion to shorten the wording and match the software terminology:

```suggestion
                {% set helptip_severity = 'Leave empty to disable log alerts by email.'|trans({}, 'Admin.Advparameters.Help') ~ ' ' ~ 'Or enter the recipients of these emails in the following field.'|trans({}, 'Admin.Advparameters.Help') %}
```